bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/ZJets_LO_MLM/ZJetsToNuNu_HT-100to200/ZJetsToNuNu_HT100to200_proc_card.dat

```diff
@@ -0,0 +1,41 @@
+#************************************************************
+#*                        MadGraph 5                        *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 1.5.11                2013-06-21         *
+#*                                                          *
+#*    The MadGraph Development Team - Please visit us at    *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph 5                *
+#*                                                          *
+#*     run as ./bin/mg5  filename                           *
+#*                                                          *
+#************************************************************
+
+import model sm-ckm_no_b_mass
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+# Define multiparticle labels
+define p u c s d b u~ c~ s~ d~ b~ g
```

Comment:
I am not sure I understand this declaration. Is an equal sign missing?
@kdlong @qliphy do you know?

Reply:
@perrozzi if I remember correctly, we do not need to redefine p and j for this model right? it already includes 5f? Thanks.

Reply:
I am puzzled because the first definition is 4F (b missing) and the second declaration doesn't have =
so I am not sure what is going on (also for the previous requests...)
In any case yes, you can remove the proton and jet definitions